File: test/create_ferm_pos.py
#!/usr/bin/python
import json

fermenters={
    1:{"pos":1,"gpio":6,"temp_serial":"28-0417711245ff"}, 
    2:{"pos":2,"gpio":5},
    3:{"pos":3,"gpio":10}, 
    4:{"pos":4,"gpio":22}, 
    5:{"pos":5,"gpio":27}, 
    6:{"pos":6,"gpio":17}, 
    7:{"pos":7,"gpio":9}, 
    8:{"pos":8,"gpio":11}
    }


for fermenter_pos in fermenters:
    print(fermenter_pos)

with open('../ferm_pos.json','w') as outfile:
    json.dump(fermenters, outfile)

File: test/create_ferm_pos.php
<?php
$pos[1]=array("gpio"=>26,"temp_serial"=>"28-041771124ff");
$pos[2]=array("gpio"=>19);
$pos[3]=array("gpio"=>10);
$pos[4]=array("gpio"=>22);
$pos[5]=array("gpio"=>27);
$pos[6]=array("gpio"=>17);
$pos[7]=array("gpio"=>9);
$pos[8]=array("gpio"=>7); 
file_put_contents("/opt/fermenters/ferm_pos",json_encode($pos));
?>
